scripts/wrappers/status.py

```diff
@@ -208,26 +209,29 @@ def ha_cluster_formed(info):
     yaml_short = args.yaml
 
     if wait_ready:
-        isReady = wait_for_ready(timeout)
+        is_ready = wait_for_ready(timeout)
+        if not is_ready:
```

Comment:
Users are probably already parsing this command's output so I would not change that. Instead I would just change the return code in the case of is_ready != true

Reply:
do you mean `print("Cluster is not ready after {} seconds.".format(timeout))` ?

I do not think we have any message now. And I believe it will be good to emphasize that behaviour has changed and now we return non-zero

Reply:
It is not ok to change the output of a script because users may be already parsing it in a very specific way. Here is an example of how this could go wrong. Let's assume a user has asked `--format yaml` and the `wait_for_ready` times out. Please do not change the out put of the status command.